tlscanary/template/js/report_page.js

```diff
@@ -27,19 +27,18 @@ function makeChartTab(uriList, fieldName) {
 
 function makeFieldControl(fieldName) {
   var html = "";
-  html += "<h3>Field:&nbsp;&nbsp;<select id=\"fieldNames\" name=\"fieldNames\" >";
+  html += "<h3>Field:&nbsp;&nbsp;<select id='fieldNames' name='fieldNames' >";
```

Comment:
Same hardquote question here. Also trailing whitespace before the closing bracket.